Tutorials/Creating-a-Custom-Dashboard/index.md

```diff
@@ -222,17 +254,17 @@ var user = userService.getCurrentUser().then(function (user) {
 Finally we can now update our view to incorporate the current user's name in our Welcome Message:
```

Comment:
Above this, there's a **Tip**
Shouldn't this be in note form instead? 🤔

Reply:
Good call, will add it as one of our custom tips instead: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips